static/createClub-script.js

```diff
@@ -0,0 +1,90 @@
+// JavaScript file for create club page
+
+
+
+var app = angular.module('createClubApp', []);
+app.controller('createClubControl', function($scope) {
+
+    $scope.message = "Register your club here!";
+
+    $scope.createClub = function(clubName, description, category, location, usualTime) {
+
+	// are all fields filled in?
+	if (clubName == undefined ||
+	    clubName == "" ||
+	    description == undefined ||
+	    description == "" ||
+	    category == undefined ||
+	    category == "" ||
+	    location == undefined ||
+	    location == "" ||
+	    usualTime == undefined ||
+	    usualTime == "") {
+	    
+	    $scope.message = "All fields required."
+	}
+
+	
+	// form fields seem good from the surface...
+	// check w/ database to see if valid registration...
+	else {
+
+	    var attendSchoolId;
+	    var getOut = false;
+	    $.getJSON('http://localhost:5000/getLogin', function(data) {
+
+		attendSchoolId = data.loggedinSchoolId;
+
+	    })
+		.done( function(data) {
+		    
+		    $.getJSON('http://localhost:5000/clubs', function(data) {
```

Comment:
hm let's not do this.... i think the workflow for this function should just be to call the `/club` service with all the things, then you're backend should respond with with a `409` http code, and if your frontend sees a 409 it displays whatever you want when that happens.... 
I would recommend punting on this work rn... I think all you really want here is that the club gets created or already exists, then redirect the user to the new (or existing) club page.... just make an issue for the work to do special sauce if hte club already exists